edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Docker.E2E.Test/AgentTests.cs

```diff
@@ -107,7 +107,7 @@ public async Task AgentStartsUpModules(TestConfig testConfig)
 
                 // Initialize docker configuration for this module.
                 DockerConfig dockerConfig = testConfig.ImageCreateOptions != null
-                    ? new DockerConfig(testConfig.Image, testConfig.ImageCreateOptions)
+                    ? new DockerConfig(testConfig.Image, testConfig.ImageCreateOptions, null)
```

Comment:
This maybe addressed in future PR: I am sure you can tell we need to create a Notary server of our own here and add a test case here for agent with notary enabled. It will be good to test with 3 trusted modules E2E  - edgeAgent, edgeHub and simulatedTempSensor.